src/components/lesson-metadata-modal.js

```diff
@@ -10,24 +10,30 @@ import React, { useState, useRef, useEffect } from 'react';
 import LanguageSelect from './localization/language-select.js';
 import { useSessionAwareApiClient } from '../ui/api-helper.js';
 import LessonApiClient from '../api-clients/lesson-api-client.js';
+import { lessonMetadataShape } from '../ui/default-prop-types.js';
 import { useDateFormat, useLanguage } from './language-context.js';
 
 const FormItem = Form.Item;
 
 const logger = new Logger(import.meta.url);
 
+export const LESSON_MODAL_MODE = {
+  create: 'create',
+  update: 'update'
+};
+
```

Comment:
I put the comment here, although I mean line 131 (but that one is not changed, so Github doesn't let me put a comment there). The title of the modal should not always be `t('newLesson')` when `mode === LESSON_MODAL_MODE.update`

Reply:
Nice catch, than you, corrected!